lib/command/open.rb

```diff
@@ -25,9 +25,9 @@ def call
       workload = config.options[:workload] || config[:one_off_workload]
       data = cp.fetch_workload!(workload)
       url = data["status"]["endpoint"]
-      opener = `which xdg-open open`.split("\n").grep_v("not found").first
+      opener = Shell.cmd("which xdg-open open")[:output].split("\n").grep_v("not found").first
 
-      exec %(#{opener} "#{url}")
+      Kernel.exec %(#{opener} "#{url}")
```

Comment:
Same here, you see, you tend use string interpolations for no any reason even with `Kernel.exec` (not only `Shell.cmd`). Should be just `Kernel.exec(opener, url)` 😉